sources/engine/Xenko.Graphics/WindowsMixedReality/WindowsMixedRealityGraphicsPresenter.cs

```diff
@@ -0,0 +1,151 @@
+#if XENKO_GRAPHICS_API_DIRECT3D11 && XENKO_PLATFORM_UWP
+
+using System;
+using System.Runtime.InteropServices;
+using SharpDX.Direct3D11;
+using Windows.Graphics.DirectX.Direct3D11;
+using Windows.Graphics.Holographic;
+using Windows.UI.Core;
+
+namespace Xenko.Graphics
+{
+    public class WindowsMixedRealityGraphicsPresenter : GraphicsPresenter
```

Comment:
Having that class in `Xenko.Graphics` forces to have dependency on `Windows.Graphics.Holographic` for all UWP projects (even those that are not targeting such device. I think we should come up with a better design.

Reply:
Yes and ideally it should all be doable as a plugin, but there is unfortunately quite a bunch of non-trivial extensions point to add, could be a lot of work (it affects renderer code and game creation).